query/physicalplan/filter.go

```diff
@@ -198,6 +220,22 @@ type OrExpr struct {
 	Right BooleanExpression
 }
 
+func (a *OrExpr) EvalParquet(rg parquet.RowGroup, in [][]parquet.Value) (*Bitmap, [][]parquet.Value, error) {
+	left, lout, err := a.Left.EvalParquet(rg, in)
```

Comment:
If left is true, we can exit with evaling the right side. This helps be more efficient.

Reply:
Ah nevermind, these are bitmaps